test/CommonModuleInit.cs

```diff
@@ -2,7 +2,29 @@
 public static class CommonModuleInit
 {
     [ModuleInitializer]
-    public static void Init() =>
+    [SuppressMessage("Usage", "CA2255:The \'ModuleInitializer\' attribute should not be used in libraries")]
+    public static void Init()
+    {
+#if TRIMMABLE
+        PathInfo DeriveAotPathInfo(string sourceFile, string projectDirectory, Type type, MethodInfo method)
+        {
+            var uniqueForAot = method.GetCustomAttribute<UniqueForAotAttribute>()
+                            ?? type.GetCustomAttribute<UniqueForAotAttribute>();
+            var derivedMethodName = (uniqueForAot != null)
+                ? method.Name + ".AOT"
+                : method.Name;
+            return new PathInfo(
+                directory: projectDirectory,
```

Comment:
This seems wrong - all ".AOT.verified" files are put in the root test directory instead of next to tests.

Reply:
example: ContextCopyTests